google_cse.module

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @file
+ * Display a Google Custom Search Engine (CSE) on your site.
+ */
+
+/**
+ * Get the relevant language to use for the search.
+ *
+ * @return string
+ *   The language.
+ */
+function google_cse_language() {
+  $language = \Drupal::languageManager()->getCurrentLanguage()->getId();
+  $local_hl = \Drupal::config('search.page.google_cse_search')->get('configuration')['locale_hl'];
+  $hl = \Drupal::config('search.page.google_cse_search')->get('configuration')['hl'];
```

Comment:
use to minimize usage of `\Drupal`
`$config = \Drupal::config('search.page.google_cse_search');`